corpus/extracted/0006-ohsnap-01/fil_7_storageminer/ReportConsensusFault/18/ext-0007-fil_7_storageminer-ReportConsensusFault-18-1.json

```
@@ -1,7 +1,8 @@
 {
   "class": "message",
   "selector": {
-    "min_protocol_version": "ohsnap"
+    "min_protocol_version": "ohsnap",
+    "consensus_fault": true
```

nit: can we make it more expressive in terms of the requirement for the runner?

```suggestion
    "requires:consensus_fault_extern": true
```

---

I'm also changing the value to a string to match the schema.